src/System.Net.Sockets/src/System/Net/Sockets/SendPacketsElementFlags.Unix.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.Sockets
+{
+    // NOTE: these values are arbitrary. The APIs that consume SendPacketsElement values are not
+    // supported on *nix.
+    internal enum SendPacketsElementFlags
+    {
+        File = 0,
```

Comment:
Shouldn't these values be defined somewhere in Interop similarly to the Windows version?